### PR TITLE
:bug: fix all hitbox

### PR DESCRIPTION
**Pull request by Quentintnrl**
fix hitbox

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -28,6 +28,8 @@
 #include "blocks/collectible/diamondRed.hpp"
 #include "blocks/platforms/pushable/Crate.hpp"
 
+#include <iostream>
```
Comment (eliotttesnier) [inline src/main.cpp:31]: Chef?

**Review by eliotttesnier — CHANGES_REQUESTED** [review]

File: src/Level.cpp [hunk 2]
```diff
@@ -10,6 +10,8 @@
 #include "Level.hpp"
 #include "Game.hpp"
 
+#include <iostream>
```
Comment (Nolfews) [inline src/Level.cpp:13]: delete this

**Review by eliotttesnier — APPROVED** [review]

**Review by Nolfews — CHANGES_REQUESTED** [review]
include

**Review by SamGave07 — APPROVED** [review]
Bonne PR et court message

**Review by Nolfews — APPROVED** [review]
:+1:  nice pr

**Review by RenardFute — APPROVED** [review]

**Pull request merged**